test/functional/autocmd/helpfile_spec.lua

```diff
@@ -0,0 +1,22 @@
+local helpers = require('test.functional.helpers')(after_each)
```

Comment:
name the file `filetype_spec.lua`, this way more tests related to `FileType` could be added later.

Reply:
ok done